tests/ert3/console/integration/test_cli.py

```diff
@@ -354,6 +314,60 @@ def test_cli_run_presampled(tmpdir):
 
     coeff0 = ert3.storage.get_variables(workspace, "coefficients0")
     assert 1000 == len(coeff0)
+    for real_coeff in coeff0:
+        assert sorted(("a", "b", "c")) == sorted(real_coeff.keys())
+        for val in real_coeff.values():
+            assert isinstance(val, float)
+    _assert_distribution(workspace, "presampled_evaluation_big", "gaussian", coeff0)
+
+
+def test_cli_uniform_distribution(tmpdir):
+    workspace = tmpdir / _POLY_WORKSPACE_NAME
+    shutil.copytree(_POLY_WORKSPACE, workspace)
+    workspace.chdir()
+
+    args = ["ert3", "init"]
+    with unittest.mock.patch.object(sys, "argv", args):
+        ert3.console.main()
+
+    args = [
+        "ert3",
+        "record",
+        "sample",
+        "uniform_coefficients",
+        "uniform_coefficients0",
+        "1000",
+    ]
+    with unittest.mock.patch.object(sys, "argv", args):
+        ert3.console.main()
+
+    coeff0 = ert3.storage.get_variables(workspace, "uniform_coefficients0")
+    assert 1000 == len(coeff0)
```

Comment:
Any reason this part of the coefficient checking is not moved into the `_assert_distribution`? Both currently expects the coefficients to be named `a`, `b` and `c`.